pkg/internal/templates.go

```diff
@@ -140,6 +153,7 @@ type wrapped{{.Name}} struct {
 	delegate {{.PkgName}}{{.Version}}.{{.Name}}Interface
 }
 
+// checkCluster returns a scoped context.
```

Comment:
This seems misleading - `check*` implies some checking is happening, no?

Reply:
Ah yeah! Modified it